BeardedSpice/UI/BSVolumeWindowController.m

```diff
@@ -0,0 +1,158 @@
+//
+//  BSVolumeWindowController.m
+//  BeardedSpice
+//
+//  Created by Roman Sokolov on 08.05.17.
+//  Copyright © 2017 BeardedSpice. All rights reserved.
+//
+
+#import "BSVolumeWindowController.h"
+
+#define BASE_FONT_NAME              @"HelveticaNeue-Medium"
+#define BASE_FONT_SIZE              36.0f
+
+@implementation BSVolumeWindowController {
+    NSTimer *_hideTimer;
+}
+
+
+static BSVolumeWindowController *singletonBSVolumeWindowController;
```

Comment:
Unless the var `singletonBSVolumeWindowController` is used elsewhere (which it shouldn't be), can we keep this static variable in the `singleton` method please?